### ISSUE TITLE
Multiple reports for one object/model

### ISSUE THREAD

**Issue by GummiHai** [WilldooIT/Pentaho-reports-for-OpenERP#109]
Hi,
I need multiple reports for one model.
Example:
In sale.order model I like to have a report with company logo and a second one without the logo and maybe a third one with different design.
It seems that only the last created report is used - however the menu items are created.
I'm using OpenERP 7.

Thanks for help.

**Comment by richard-willdooit** [comment]
@GummiHai 

Could you clarify please - You have created one report (prpt file) or multiple?
How many OpenERP actions have you created - one or three?
If there is only one prpt file, what is the trigger that decides which report to run?
If you have created three OpenERP actions, have you given each one a unique service name?

Please elaborate on these points where you can.  From what you have told me, I am suspecting you have not given the actions unique service names.

Richard

**Comment by GummiHai** [comment]
I have created two different report files. One with a logo and the second one where the logo is not visible. I just created two actions at "Configuration->Actions->Reports". For every action I'd uploaded a different report file (ex. SaleOrder.prpt and SaleOrderNoLogo.prpt).

@richard-willowit : You are my treasure :) Your hint with the service name was the solution!!!

For the second report I have to put just "sale.order.nologo" or something else, then it works!

Thank you very much.

Regards
Juergen
